### PR TITLE
4.0 Tweaks

### PR DESCRIPTION
**Pull request by ZevEisenberg**

### CODE REVIEW

File: Example-iOS/AppDelegate.swift [hunk 1]
```diff
@@ -24,12 +24,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         guard let traitCollection = window?.traitCollection else {
             fatalError("There should be a traitCollection available before calling this method.")
         }
-        let titleStyle = StringStyle.style(
+        let titleStyle = StringStyle(parts:
```
Comment (dostrander) [inline Example-iOS/AppDelegate.swift]: #? Why do we even need to have a named param (`parts:`). I feel it's obvious with the `dot` syntax we have going on like `.font().color()`

File: BonMot.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -62,7 +62,7 @@
 		ABA75DD61D99E10400B64705 /* StringStyle.swift in Sources */ = {isa = PBXBuildFile; fileRef = ABCD3D521D96F02F00273936 /* StringStyle.swift */; };
 		ABA75DD81D99E10400B64705 /* Special.swift in Sources */ = {isa = PBXBuildFile; fileRef = ABCD3D5B1D96F02F00273936 /* Special.swift */; };
 		ABA75DD91D99E10400B64705 /* NamedStyles.swift in Sources */ = {isa = PBXBuildFile; fileRef = ABCD3D5E1D96F02F00273936 /* NamedStyles.swift */; };
-		ABA75DDB1D99E10400B64705 /* StringStylePart.swift in Sources */ = {isa = PBXBuildFile; fileRef = ABCD3D531D96F02F00273936 /* StringStylePart.swift */; };
+		ABA75DDB1D99E10400B64705 /* StylePart.swift in Sources */ = {isa = PBXBuildFile; fileRef = ABCD3D531D96F02F00273936 /* StylePart.swift */; };
```
Comment (dostrander) [inline BonMot.xcodeproj/project.pbxproj]: :+1:

**Comment by dostrander** [comment]
Still in progress… just wanted to at least get these comments in

File: Example-iOS/AppDelegate.swift (hunk 1 above)
Comment (ZevEisenberg) [inline Example-iOS/AppDelegate.swift]: @dostrander if you can delete the parameter name and get it to compile on Swift 2.3 (change the setting in the Xcode project), I'd love to see it. I couldn't get it to work.

File: .swift-version [hunk 3]
```diff
@@ -1 +1 @@
-3.0
+3.0.1
```
Comment (KingOfBrian) [inline .swift-version]: I'd stick with `3.0`

File: BonMot.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -1484,7 +1484,7 @@
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				MTL_ENABLE_DEBUG_INFO = NO;
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
-				SWIFT_VERSION = 3.0;
+				SWIFT_VERSION = 3.0.1;
```
Comment (KingOfBrian) [inline BonMot.xcodeproj/project.pbxproj]: This should stay `3.0` too. Won't compile on Xcode 8.

File: .swift-version (hunk 3 above)
Comment (ZevEisenberg) [inline .swift-version]: I think I was getting a warning in Xcode 8.1 when I did.

File: BonMot.xcodeproj/project.pbxproj (hunk 4 above)
Comment (ZevEisenberg) [inline BonMot.xcodeproj/project.pbxproj]: Xcode 8.1 was showing a warning, but I guess that's better than an error, so I'll revert it.

File: Example-iOS/AppDelegate.swift (hunk 1 above)
Comment (KingOfBrian) [inline Example-iOS/AppDelegate.swift]: We figured out the magic to drop `parts:` on 2.3!

File: Sources/Composable.swift [hunk 5]
```diff
@@ -42,10 +42,10 @@ public extension Composable {
     ///
     /// - parameter parts: The style parts to decorate with
     /// - returns: A new NSAttributedString
-    public func styled(with parts: StringStylePart...) -> NSAttributedString {
+    public func styled(with parts: StylePart...) -> NSAttributedString {
         var style = StringStyle()
         for part in parts {
-            style.update(stringStylePart: part)
+            style.update(StylePart: part)
```
Comment (ZevEisenberg) [inline Sources/Composable.swift]: Oops. Fixing. Nice catch.

**Pull request merged**